CHANGELOG.md

```diff
@@ -3,6 +3,34 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [v1.1.1][v1.1.1] - 31-Oct-2021
+
+### Milestone: [catapult-server@v1.0.2.0][catapult-server@v1.0.2.0]
```

Comment:
is it possible to remove the milestone, or perhaps have a min client version eg. >= 1.0.2.0?